omero/sysadmins/server-upgrade.txt

```diff
@@ -263,6 +263,27 @@ database which can both save disk space and speed up access times.
     $ psql -h localhost -U **db_user** **omero_database** -c 'REINDEX DATABASE "**omero_database**" FORCE;'
     $ psql -h localhost -U **db_user** **omero_database** -c 'VACUUM FULL VERBOSE ANALYZE;'
 
+Patch database functions (optional)
+"""""""""""""""""""""""""""""""""""
+
+An additional database script is provided for fixing an error that
+affects *only* systems that were once upgraded from OMERO 4.4. More
+recent OMERO installations should skip this patching step.
+
+The script patches OMERO 5.1 and 5.2 databases, runs quickly, and can be
+applied anytime, even under a running OMERO server. Although the problem
+has been sufficiently asymptomatic to go unnoticed for some time we
+nonetheless recommend that the patch is applied promptly.
+
+.. parsed-literal::
+
+    $ psql -h localhost -U **db_user** **omero_database** < sql/psql/|current_dbver|/OMERO5.2-5.1-format-syntax-patch.sql
+
+An unpatched system may exhibit serious problems if run with PostgreSQL
+9.5. OMERO 5.2 does *not* support PostgreSQL 9.5 and the upgrade script
+for OMERO 5.3 *will* ensure that these PL/pgSQL functions are patched
+regardless of if you used this provided script before that upgrade.
```

Comment:
would this sentence read better as '…regardless of *whether* you used…'? 'of if' is a bit difficult to parse